### DISCUSSION TITLE
Getting Matchup scores throughout the week (Basketball)

### DISCUSSION THREAD

**Discussion by ThomasAdriaanse** [cwendt94/espn-api#594] [ESPN API] [answered]
Is there any way to get the matchup score of each of the two teams for each day of the matchup so far? 

If not I think this would be a cool feature, nice for graphing the scores over the matchup.

**Comment by cwendt94** [comment, marked as answer]
You should be able to get this information with box_score functionality. See the [box_score](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) at the bottom of the league wiki page. If you pass in scoring period that represents the day matchup.

**Reply by ThomasAdriaanse** [reply]
Thanks I didn't notice those settings, I tested it out and it works great for me
